backend/api/Database/Context/FlotillaDbContext.cs

```diff
@@ -26,6 +27,7 @@ public FlotillaDbContext(DbContextOptions options) : base(options)
         public DbSet<DefaultLocalizationPose> DefaultLocalizationPoses => Set<DefaultLocalizationPose>();
         public DbSet<AccessRole> AccessRoles => Set<AccessRole>();
         public DbSet<UserInfo> UserInfos => Set<UserInfo>();
+        public DbSet<EchoTagInspectionMetadata> EchoTagInspectionMetadata => Set<EchoTagInspectionMetadata>();
```

Comment:
Avoid using Echo naming

Reply:
This object is linked to the EchoTag model which already exists in the codebase

Reply:
Not in the database, only in the service and controller models